child_sync_wp/models/child_compassion.py

```diff
@@ -28,8 +28,8 @@ class CompassionChild(models.Model):
     @api.multi
     def add_to_wordpress(self, company_id=None):
         # Solve the encoding problems on child's descriptions
-        reload(sys)
-        sys.setdefaultencoding('UTF8')
+        importlib.reload(sys)
+        # sys.setdefaultencoding('UTF8')
```

Comment:
I would try to remove this completely, I think encoding problems have being improved in python3 so by default it should work.